unstructured/file_utils/metadata.py

```diff
@@ -150,3 +153,43 @@ def _get_exif_datetime(exif_dict: Dict[str, Any], key: str) -> Optional[datetime
     # using the standard EXIF datetime format
     except ValueError:
         return None
+
+
+def apply_lang_metadata(
+    elements: Union[Iterable[Element], List[Element]],
```

Comment:
`list` is iterable, so this can be simplified to just `Iterable[Element]`.

Reply:
I'm getting this error in the checks if I remove that (in an number of the partitioners, not just `xlsx`):
`unstructured/partition/xlsx.py:176: error: Incompatible types in assignment (expression has type "Iterator[Element]", variable has type "List[Element]")  [assignment]`

Should I add it back because of the linting error?